engine/runtime/src/main/scala/org/enso/compiler/SerializationManager.scala

```diff
@@ -617,43 +614,6 @@ final class SerializationManager(
       )
       .asScala
   }
-
-  /** Generate the documentation for the given suggestion.
-    *
-    * @param suggestion the initial suggestion
-    * @return the suggestion with documentation fields set
-    */
-  private def generateDocumentation(suggestion: Suggestion): Suggestion =
```

Comment:
This method was a source of major slowdowns (as mentioned for example in #5027). There doesn't seem to be a better way to speed some code than deleting it! Great.